### PR TITLE
Include legacy versions for upgrading

### PR DESCRIPTION
**Pull request by jesperlange**
A lot of people are still on v8, and although we no longer support it I still believe it's good to have included how you can upgrade your project manually, as the steps are not the same compared to v9+

Here, I have made a little dropdown to include them, its not intrusive, however still available info for those that need it.

Not sure about the wording on the "Summary" though.

### CODE REVIEW

File: umbraco-cloud/product-upgrades/manual-upgrades/manual-upgrade-deploy.md [hunk 1]
```diff
@@ -41,6 +41,29 @@ When the command completes, open the `.csproj` file to make sure the package ref
 
 Make sure that everything works on the local clone and that you can **run the project without any errors**.
 
+<details>
+
+<summary>Upgrading Deploy Legacy Version 7 & 8</summary>
+
```
Comment (jonat123) [inline umbraco-cloud/product-upgrades/manual-upgrades/manual-upgrade-deploy.md:47]:
```suggestion
  If you are on Umbraco 7 or Umbraco 8, follow the steps below to manually upgrade Umbraco Deploy to a later version of your project.
```

File: umbraco-cloud/product-upgrades/manual-upgrades/manual-upgrade-deploy.md [hunk 2]
```diff
@@ -41,6 +41,29 @@ When the command completes, open the `.csproj` file to make sure the package ref
 
 Make sure that everything works on the local clone and that you can **run the project without any errors**.
 
+<details>
+
+<summary>Upgrading Deploy Legacy Version 7 & 8</summary>
```
Comment (jonat123) [inline umbraco-cloud/product-upgrades/manual-upgrades/manual-upgrade-deploy.md:46]:
```suggestion
<summary>Manually Upgrade Umbraco Deploy Legacy versions 7 and 8</summary>
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@jesperlange Looks good!

Just made two suggestions :D

**Comment by jesperlange** [comment]
awesome man, I've included your suggestions

File: umbraco-cloud/product-upgrades/manual-upgrades/manual-upgrade-deploy.md [hunk 3]
```diff
@@ -41,6 +41,29 @@ When the command completes, open the `.csproj` file to make sure the package ref
 
 Make sure that everything works on the local clone and that you can **run the project without any errors**.
 
+<details>
+
+<summary>Manually Upgrade Umbraco Deploy Legacy versions 7 and 8</summary>
+  If you are on Umbraco 7 or Umbraco 8, follow the steps below to manually upgrade Umbraco Deploy to a later version of your project.
```
Comment (jonat123) [inline umbraco-cloud/product-upgrades/manual-upgrades/manual-upgrade-deploy.md:47]:
```suggestion

  If you are on Umbraco 7 or Umbraco 8, follow the steps below to manually upgrade Umbraco Deploy to a later version of your project.
  
```

**Review by jonat123 — APPROVED** [review]
Looks good!

**Pull request merged**